recipes/matplotview/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "matplotview" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/matplotview-{{ version }}.tar.gz
+  sha256: debfffa8d27db435a13b1f3ef10d8b88b119aca42cc37c0de05934af97c1e45f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools >=42
+    - wheel
+  run:
+    - python >=3.7
+    - matplotlib-base >=3.5.1
+    - numpy
+
+test:
+  imports:
+    - matplotview
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/matplotlib/matplotview
+  summary: A library for creating lightweight views of matplotlib axes.
+  license: BSD-4-Clause
```

I think it's actually
```suggestion
  license: PSF-2.0
```

---

@marcelotrevisani grayskull is consistently using BSD-4-Clause for PSF-2.0 licenses (third time I see that happening). I'll investigate and open an issue upstream.